util/globals.py

```diff
@@ -48,6 +48,30 @@ def get_macros(self, ioc_name):
                 macros[key] = value
         return macros
 
+    def get_values_of_macro(self, macro_name):
+        """
+        Get the values of all macros of a given name.
+        :param macro_name: The name of the macro to search for
+        :return:  A dictionary of all instances of the matching values and the values.
+        """
+        lines = self.get_lines()
+        macros = dict()
+        for line in lines:
+            if line.__contains__(macro_name):
+                line = line.split("__")[1]
+                key, value = line.split("=")
+                macros[key] = value
+        return macros
+
+    def is_in_sim_mode(self):
```

Comment:
This is checking whether *any* IOC is set to simulation mode within the globals file. I think this should be made clearer from the method name. Something like `is_any_ioc_in_sim_mode`?